<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> publications | Díaz Cruz Lab </title> <meta name="author" content="Gustavo Díaz Cruz"> <meta name="description" content="publications by categories in reversed chronological order. generated by jekyll-scholar."> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" href="/assets/css/scholar-icons.css?62b2ac103a88034e6882a5be5f3e2772"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="/assets/img/icon.png?a43ed57d408b50396385df9db7dbd87c"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://gdiazcruz.github.io/publications/"> <script src="/assets/js/theme.js?9a0c749ec5240d9cda97bc72359a72c0"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>
    initTheme();
  </script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> Díaz Cruz Lab </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about </a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/projects/">projects </a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv </a> </li> <li class="nav-item "> <a class="nav-link" href="/people/">people </a> </li> <li class="nav-item dropdown "> <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">submenus </a> <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown"> <a class="dropdown-item " href="/books/">bookshelf</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/blog/">blog</a> </div> </li> <li class="nav-item"> <button id="search-toggle" title="Search" onclick="openSearchModal()"> <span class="nav-link">ctrl k <i class="ti ti-search"></i></span> </button> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications</h1> <p class="post-description">publications by categories in reversed chronological order. generated by jekyll-scholar.</p> </header> <article> <script src="/assets/js/bibsearch.js?1bc438ca9037884cc579601c09afd847" type="module"></script> <p><input type="text" id="bibsearch" spellcheck="false" autocomplete="off" class="search bibsearch-form-input" placeholder="Type to filter"></p> <div class="publications"> <h2 class="bibliography">2024</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Diaz-Cruz2024" class="col-sm-8"> <div class="title">Exploring the specialized metabolome of the plant pathogen Streptomyces sp. 11-1-2</div> <div class="author"> Gustavo A. Díaz-Cruz, and Dawn R.D. Bignell </div> <div class="periodical"> <em>Scientific Reports</em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/s41598-024-60630-5" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Streptomyces bacteria are notable for producing chemically diverse specialized metabolites that exhibit various bioactivities and mediate interactions with different organisms. Streptomyces sp. 11-1-2 is a plant pathogen that produces nigericin and geldanamycin, both of which display toxic effects against various plants. Here, the ‘One Strain Many Compounds’ approach was used to characterize the metabolic potential of Streptomyces sp. 11-1-2. Organic extracts were prepared from 11-1-2 cultures grown on six different agar media, and the extracts were tested in antimicrobial and plant bioassays and were subjected to untargeted metabolomics and molecular networking. Most extracts displayed strong bioactivity against Gram-positive bacteria and yeast, and they exhibited phytotoxic activity against potato tuber tissue and radish seedlings. Several known specialized metabolites, including musacin D, galbonolide B, guanidylfungin A, meridamycins and elaiophylin, were predicted to be present in the extracts along with closely related compounds with unknown structure and bioactivity. Targeted detection confirmed the presence of elaiophylin in the extracts, and bioassays using pure elaiophylin revealed that it enhances the phytotoxic effects of geldanamycin and nigericin on potato tuber tissue. Overall, this study reveals novel insights into the specialized metabolites that may mediate interactions between Streptomyces sp. 11-1-2 and other bacteria and eukaryotic organisms.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2022</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Diaz-Cruz2022" class="col-sm-8"> <div class="title">Changes in the phyllosphere and rhizosphere microbial communities of soybean in the presence of pathogens</div> <div class="author"> Gustavo A. Díaz-Cruz, and Bryan J. Cassone </div> <div class="periodical"> <em>FEMS Microbiology Ecology</em>, 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1093/femsec/fiac022" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Soybean (Glycine max L.) is host to an array of foliar- A nd root-infecting pathogens that can cause significant yield losses. To provide insights into the roles of microorganisms in disease development, we evaluated the bacterial and fungal communities associated with the soybean rhizosphere and phyllosphere. For this, leaf and soil samples of healthy, Phytophthora sojae-infected and Septoria glycines-infected plants were sampled at three stages during the production cycle, and then subjected to 16S and Internal Transcribed Spacer (ITS) amplicon sequencing. The results indicated that biotic stresses did not have a significant impact on species richness and evenness regardless of growth stage. However, the structure and composition of soybean microbial communities were dramatically altered by biotic stresses, particularly for the fungal phyllosphere. Additionally, we cataloged a variety of microbial genera that were altered by biotic stresses and their associations with other genera, which could serve as biological indicators for disease development. In terms of soybean development, the rhizosphere and phyllosphere had distinct microbial communities, with the fungal phyllosphere most influenced by growth stage. Overall, this study characterized the phyllosphere and rhizosphere microbial communities of soybean, and described the impact of pathogen infection and plant development in shaping these bacterial and fungal communities.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Diaz-Cruz2022a" class="col-sm-8"> <div class="title">Nigericin and Geldanamycin Are Phytotoxic Specialized Metabolites Produced by the Plant Pathogen Streptomyces sp. 11-1-2</div> <div class="author"> Gustavo A. Díaz-Cruz, Jingyu Liu, Kapil Tahlan, and Dawn R D Bignell </div> <div class="periodical"> <em>Microbiology Spectrum</em>, Apr 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1128/spectrum.02314-21" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Plant pathogens use a variety of mechanisms, including the production of phytotoxic specialized metabolites, to establish an infection of host tissue. Although thaxtomin A is considered the key phytotoxin involved in the development of potato scab disease, there is increasing evidence that other phytotoxins can play a role in disease development in some instances.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2021</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Diaz-Cruz2021" class="col-sm-8"> <div class="title">Amplicon sequencing reveals extensive coinfections of foliar pathogens in soybean</div> <div class="author"> Gustavo A. Díaz-Cruz, and Bryan J. Cassone </div> <div class="periodical"> <em>Plant Disease</em>, Apr 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1094/PDIS-06-20-1165-RE" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Soybean (Glycine max) is one of the most economically important crops grown in North America and in other regions worldwide. However, the plant is susceptible to a variety of foliar pathogenic microorganisms, some of which are a significant threat to production. Several molecular and serological approaches are currently available to diagnose plant pathogens, but all have limitations including their capability to accurately detect coinfections of individual plants. We therefore used 16S and internal transcribed spacer amplicon sequencing to identify the suite of bacterial and fungal organisms infecting 96 soybean leaf samples collected throughout southern Manitoba, Canada, at two growth stages (V2/3 and R6). We were able to confirm the presence of pathogens previously known to our sampling regions, such as Septoria glycines, Alternaria alternata, and Pseudomonas spp. Importantly, we found that most of plants were infected by more than one putative pathogen, with 64% of V2/3 and 26% of R6 plants infected by three or more pathogens. Amplicon sequencing also indicated the presence of residual pathogens that infect crops other than soybean, as well as nonfoliar pathogens and nonpathogenic microorganisms. We discuss some of the benefits and drawbacks of using amplicon sequencing to detect foliar pathogens of soybean.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2019</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Diaz-Cruz2019" class="col-sm-8"> <div class="title">Applications of next-generation sequencing for large-scale pathogen diagnoses in soybean</div> <div class="author"> Gustavo A. Díaz-Cruz, Charlotte M. Smith, Kiana F. Wiebe, Sachi M. Villanueva, Adam R. Klonowski, and Bryan J. Cassone </div> <div class="periodical"> <em>Plant Disease</em>, Apr 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1094/PDIS-05-18-0905-RE" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Soybean (Glycine max) has become an important crop in Manitoba, Canada, with a 10-fold increase in dedicated acreage over the past decade. Given the rapid increase in production, scarce information about foliar diseases present in the province has been recorded. In order to describe the foliar pathogens affecting this legume, we harnessed next-generation sequencing (NGS) to carry out a comprehensive survey across Manitoba in 2016. Fields were sampled during the V2/3 (33 fields) and R6 (70 fields) growth stages, with at least three symptomatic leaves per field collected and subjected to RNA sequencing. We successfully detected several bacteria, fungi, and viruses known to infect soybean, including Pseudomonas savastanoi pv. glycinea, Septoria glycines, and Peronospora manshurica, as well as pathogens not previously identified in the province (e.g., Pseudomonas syringae pv. tabaci, Cercospora sojina, and Bean yellow mosaic virus). For some microorganisms, we were able to disentangle the different pathovars present and/or assemble their genome sequence. Since NGS generates data on the entire flora and fauna occupying a leaf sample, we also identified residual pathogens (i.e., pathogens of crops other than soybean) and multiple species of arthropod pests. Finally, the sequence information produced by NGS allowed for the development of polymerase chain reaction-based diagnostics for some of the most widespread and important pathogens. Although there are many benefits of using NGS for large-scale plant pathogen diagnoses, we also discuss some of the limitations of this technology.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Li2019" class="col-sm-8"> <div class="title">Virulence mechanisms of plant-pathogenic Streptomyces species: An updated review</div> <div class="author"> Yuting Li, Jingyu Liu, Gustavo Díaz-Cruz, Zhenlong Cheng, and Dawn R.D. Bignell </div> <div class="periodical"> <em>Microbiology (United Kingdom)</em>, Apr 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1099/mic.0.000818" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Gram-positive Actinobacteria from the genus Streptomyces are best known for their morphological complexity and for their ability to produce numerous bioactive specialized metabolites with useful applications in human and veterinary medicine and in agriculture. In contrast, the ability to infect living plant tissues and to cause diseases of root and tuber crops such as potato common scab (CS) is a rare attribute among members of this genus. Research on the virulence mechanisms of plant-pathogenic Streptomyces spp. has revealed the importance of the thaxtomin phytotoxins as key pathogenicity determinants produced by several species. In addition, other phytotoxic specialized metabolites may contribute to the development or severity of disease caused by Streptomyces spp., along with the production of phytohormones and secreted proteins. A thorough understanding of the molecular mechanisms of plant pathogenicity will enable the development of better management procedures for controlling CS and other plant diseases caused by the Streptomyces.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2018</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Diaz-Cruz2018" class="col-sm-8"> <div class="title">First report of brome mosaic virus infecting soybean, isolated in Manitoba, Canada</div> <div class="author"> G. A. Díaz-Cruz, C. M. Smith, K. F. Wiebe, J. M. Charette, and B. J. Cassone </div> <div class="periodical"> <em>Plant Disease</em>, Apr 2018 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1094/PDIS-07-17-1012-PDN" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Diaz-Cruz2018a" class="col-sm-8"> <div class="title">A tale of survival: Molecular defense mechanisms of soybean to overcome Soybean mosaic virus infection</div> <div class="author"> Gustavo A. Díaz-Cruz, and Bryan J. Cassone </div> <div class="periodical"> <em>Physiological and Molecular Plant Pathology</em>, Apr 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1016/j.pmpp.2017.11.006" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Plant viruses represent some of the greatest contributors to crop losses worldwide. Viral disease symptoms often include mosaic leaf patterns, crinkled and yellowed leaves, plant stunting and even necrosis. As intracellular pathogens, viruses hijack host cell machinery for their own replication, which can elicit dramatic changes to their hosts at the cellular, molecular and physiological levels. Soybean mosaic virus (SMV) is a widely distributed soybean pathogen, which can cause severe stunting and yield losses to infected plants. A better understanding of the underlying interactions between SMV and soybean will aid in the establishment of disease management plans. In these studies, we implemented high throughput RNA sequencing approaches and targeted metabolite profiling to describe transcriptomic and metabolic changes occurring in soybean leaves ten days post-infection with SMV. A massive defense response was detected by the increased accumulation of transcripts associated with biotic stresses, including those involved in pathogen recognition, autophagy and defense inductors. Moreover, significant decreases in expression were detected for transcripts associated with fundamental growth and development processes, such as nutrient transport and photosynthesis. At the metabolomic level, significant changes were identified in correspondence with viral infection, particularly in amino acid concentrations. Overall, it appears as though SMV inoculated plants reroute their limited resources as a survival strategy, favoring defense over plant development.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2017</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Diaz-Cruz2017" class="col-sm-8"> <div class="title">First Complete Genome Sequence of Tobacco necrosis virus D Isolated from Soybean and from North America</div> <div class="author"> Gustavo A. Díaz-Cruz, Charlotte M Smith, Kiana F Wiebe, and Bryan J Cassone </div> <div class="periodical"> <em>Genome Announcements</em>, Aug 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1128/genomeA.00781-17" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>We present the first complete genome sequence of the tombusvirus Tobacco necrosis virus D (TNV-D) from North America, obtained from an infected soybean plant. Compared with the three other TNV-D genomes isolated from different geographic regions and host plants, its nucleotide identities were between 83% and 93%.</p> </div> </div> </div> </li></ol> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2025 Gustavo Díaz Cruz. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js?a0db7e5d5c70cc3252b3138b0c91dcaf" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?e0514a05c5c95ac1a93a8dfd5249b92e"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-mml-chtml.js" integrity="sha256-MASABpB4tYktI2Oitl4t+78w/lyA+D7b/s9GEP0JOGI=" crossorigin="anonymous"></script> <script src="/assets/js/mathjax-setup.js?a5bb4e6a542c546dd929b24b8b236dfd"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script defer src="/assets/js/progress-bar.js?2f30e0e6801ea8f5036fa66e1ab0a71a" type="text/javascript"></script> <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>
    addBackToTop();
  </script> <script type="module" src="/assets/js/search/ninja-keys.min.js?a3446f084dcaecc5f75aa1757d087dcf"></script> <ninja-keys hidebreadcrumbs noautoloadmdicons placeholder="Type to start searching"></ninja-keys> <script src="/assets/js/search-setup.js?6c304f7b1992d4b60f7a07956e52f04a"></script> <script src="/assets/js/search-data.js"></script> <script src="/assets/js/shortcut-key.js?6f508d74becd347268a7f822bca7309d"></script> </body> </html>